areas:
  area_1:
    name: Farmland 1
    crop: Wheat
    x: -12.0
    y: 9.0
    z: 2.0
    color: red
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.32
      soil_moisture_index: 0.48
      veg_health_index: 0.6
      heatwave_days: 9
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.41
      soil_moisture_index: 0.44
      veg_health_index: 0.55
      heatwave_days: 11
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.47
      soil_moisture_index: 0.4
      veg_health_index: 0.52
      heatwave_days: 13
      drought_declared: true
    - year: 2024
      rainfall_deficit: 0.54
      soil_moisture_index: 0.36
      veg_health_index: 0.48
      heatwave_days: 15
      drought_declared: true
    - year: 2025
      rainfall_deficit: 0.62
      soil_moisture_index: 0.33
      veg_health_index: 0.41
      heatwave_days: 18
      drought_declared: true
  area_2:
    name: Farmland 2
    crop: Soybean
    x: -2.0
    y: 11.5
    z: 2.0
    color: blue
    size: 12.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.12
      soil_moisture_index: 0.62
      veg_health_index: 0.78
      heatwave_days: 5
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.18
      soil_moisture_index: 0.6
      veg_health_index: 0.74
      heatwave_days: 6
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.2
      soil_moisture_index: 0.58
      veg_health_index: 0.72
      heatwave_days: 7
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.22
      soil_moisture_index: 0.56
      veg_health_index: 0.7
      heatwave_days: 7
      drought_declared: false
    - year: 2025
      rainfall_deficit: 0.24
      soil_moisture_index: 0.54
      veg_health_index: 0.69
      heatwave_days: 8
      drought_declared: false
  area_3:
    name: Farmland 3
    crop: Maize
    x: 11.0
    y: 7.5
    z: 2.0
    color: yellow
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.28
      soil_moisture_index: 0.52
      veg_health_index: 0.63
      heatwave_days: 8
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.36
      soil_moisture_index: 0.48
      veg_health_index: 0.58
      heatwave_days: 10
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.44
      soil_moisture_index: 0.45
      veg_health_index: 0.54
      heatwave_days: 12
      drought_declared: true
    - year: 2024
      rainfall_deficit: 0.55
      soil_moisture_index: 0.38
      veg_health_index: 0.49
      heatwave_days: 16
      drought_declared: true
    - year: 2025
      rainfall_deficit: 0.6
      soil_moisture_index: 0.34
      veg_health_index: 0.45
      heatwave_days: 19
      drought_declared: true
  area_4:
    name: Farmland 4
    crop: Barley
    x: -11.0
    y: 1.5
    z: 2.0
    color: purple
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.22
      soil_moisture_index: 0.58
      veg_health_index: 0.66
      heatwave_days: 6
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.27
      soil_moisture_index: 0.54
      veg_health_index: 0.61
      heatwave_days: 8
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.35
      soil_moisture_index: 0.5
      veg_health_index: 0.58
      heatwave_days: 9
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.39
      soil_moisture_index: 0.47
      veg_health_index: 0.55
      heatwave_days: 11
      drought_declared: false
    - year: 2025
      rainfall_deficit: 0.43
      soil_moisture_index: 0.44
      veg_health_index: 0.52
      heatwave_days: 12
      drought_declared: true
  area_5:
    name: Farmland 5
    crop: Vegetables
    x: 1.0
    y: -1.0
    z: 2.0
    color: orange
    size: 12.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.08
      soil_moisture_index: 0.68
      veg_health_index: 0.8
      heatwave_days: 3
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.1
      soil_moisture_index: 0.66
      veg_health_index: 0.79
      heatwave_days: 4
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.12
      soil_moisture_index: 0.65
      veg_health_index: 0.78
      heatwave_days: 4
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.15
      soil_moisture_index: 0.63
      veg_health_index: 0.77
      heatwave_days: 5
      drought_declared: false
    - year: 2025
      rainfall_deficit: 0.17
      soil_moisture_index: 0.62
      veg_health_index: 0.76
      heatwave_days: 5
      drought_declared: false
  area_6:
    name: Farmland 6
    crop: Sunflower
    x: 9.0
    y: -2.0
    z: 2.0
    color: cyan
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.25
      soil_moisture_index: 0.55
      veg_health_index: 0.64
      heatwave_days: 7
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.3
      soil_moisture_index: 0.52
      veg_health_index: 0.6
      heatwave_days: 9
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.37
      soil_moisture_index: 0.49
      veg_health_index: 0.57
      heatwave_days: 10
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.45
      soil_moisture_index: 0.45
      veg_health_index: 0.53
      heatwave_days: 12
      drought_declared: true
    - year: 2025
      rainfall_deficit: 0.48
      soil_moisture_index: 0.42
      veg_health_index: 0.5
      heatwave_days: 13
      drought_declared: true
  area_7:
    name: Farmland 7
    crop: Rice
    x: -8.0
    y: -9.5
    z: 2.0
    color: magenta
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.18
      soil_moisture_index: 0.6
      veg_health_index: 0.7
      heatwave_days: 5
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.22
      soil_moisture_index: 0.57
      veg_health_index: 0.67
      heatwave_days: 6
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.26
      soil_moisture_index: 0.55
      veg_health_index: 0.65
      heatwave_days: 7
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.31
      soil_moisture_index: 0.52
      veg_health_index: 0.62
      heatwave_days: 8
      drought_declared: false
    - year: 2025
      rainfall_deficit: 0.35
      soil_moisture_index: 0.5
      veg_health_index: 0.6
      heatwave_days: 9
      drought_declared: false
  area_8:
    name: Farmland 8
    crop: Cotton
    x: 0.5
    y: -12.0
    z: 2.0
    color: lime
    size: 12.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.35
      soil_moisture_index: 0.5
      veg_health_index: 0.58
      heatwave_days: 10
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.4
      soil_moisture_index: 0.47
      veg_health_index: 0.55
      heatwave_days: 11
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.46
      soil_moisture_index: 0.44
      veg_health_index: 0.52
      heatwave_days: 13
      drought_declared: true
    - year: 2024
      rainfall_deficit: 0.51
      soil_moisture_index: 0.41
      veg_health_index: 0.49
      heatwave_days: 14
      drought_declared: true
    - year: 2025
      rainfall_deficit: 0.58
      soil_moisture_index: 0.37
      veg_health_index: 0.46
      heatwave_days: 16
      drought_declared: true
  area_9:
    name: Farmland 9
    crop: Peanuts
    x: 8.5
    y: -11.0
    z: 2.0
    color: olive
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.29
      soil_moisture_index: 0.53
      veg_health_index: 0.61
      heatwave_days: 8
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.33
      soil_moisture_index: 0.5
      veg_health_index: 0.58
      heatwave_days: 9
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.39
      soil_moisture_index: 0.47
      veg_health_index: 0.55
      heatwave_days: 11
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.44
      soil_moisture_index: 0.44
      veg_health_index: 0.52
      heatwave_days: 12
      drought_declared: true
    - year: 2025
      rainfall_deficit: 0.5
      soil_moisture_index: 0.41
      veg_health_index: 0.49
      heatwave_days: 14
      drought_declared: true
  area_10:
    name: Farmland 10
    crop: Pulses
    x: 3.5
    y: 4.0
    z: 2.0
    color: navy
    size: 10.0
    drought_history:
    - year: 2021
      rainfall_deficit: 0.2
      soil_moisture_index: 0.58
      veg_health_index: 0.68
      heatwave_days: 6
      drought_declared: false
    - year: 2022
      rainfall_deficit: 0.24
      soil_moisture_index: 0.55
      veg_health_index: 0.65
      heatwave_days: 7
      drought_declared: false
    - year: 2023
      rainfall_deficit: 0.28
      soil_moisture_index: 0.53
      veg_health_index: 0.63
      heatwave_days: 8
      drought_declared: false
    - year: 2024
      rainfall_deficit: 0.33
      soil_moisture_index: 0.5
      veg_health_index: 0.6
      heatwave_days: 9
      drought_declared: false
    - year: 2025
      rainfall_deficit: 0.38
      soil_moisture_index: 0.48
      veg_health_index: 0.58
      heatwave_days: 10
      drought_declared: false
start_position:
  x: 0.0
  y: -20.0
  z: 2.0
num_drones: 18
exploration:
  patrol_pattern: grid
  waypoint_spacing: 2.5
  flight_altitude: 2.0
allocation:
  min_drones_per_area: 1
  max_drones_per_area: 3
  reserve_drones: 8
  measurement_noise: 0.15
  idle_measurement_noise: 0.05
  boundary_soft_margin: 0.4
